Update makefiles

WBi-Wim opened the pull request:
Untested, only for moduline 2.

WBi-Wim committed:
Added a parameter to the callback function to enable parallel building of referenced models when the Parallel computing toolbox is used.

WBi-Wim committed:
Updated template make file for the moduline 2 controllers:
- various semantic changes
- added Start dir token to locate libraries
- Added product definition for the lib of areferenced model
- Added build rule to generate library of a referenced model

WBi-Wim committed:
Added some excemptions for handling of the referenced models

SpieringsAE commented on blockset/ert_linux_target_32.tmf:
I am afraid this is the wrong file, the current linux target uses the ert_linux_target_64.tmf, this one was for the old Moduline III

WBi-Wim replied:
@SpieringsAE : I added a new commit updating this file. 
I expect it to break in the making of the sharedlib for a referenced model, but i might succeed.

WBi-Wim committed:
Updated template make file for all controllers:
- various semantic changes
- added Start dir token to locate libraries
- Added MODELREF_INC_PATH token to add includes for referenced models
- removed the EXT_SRC from the model=>target extension line. Not declared so not used?
- Added product definition for the lib of a referenced model
- Added build rule to generate library of a referenced model

SpieringsAE commented:
Merged after doing some changes through external means:  
https://github.com/GOcontroll/GOcontroll-Simulink/tree/referenced_models

Thanks very much!
